app/javascript/mastodon/features/list_timeline/index.js

```diff
@@ -120,13 +120,17 @@ export default class ListTimeline extends React.PureComponent {
     if (typeof list === 'undefined') {
       return (
         <Column>
-          <LoadingIndicator />
+          <div className='scrollable'>
```

Comment:
Is there a reason to not make LoadingIndicator and MissingIndicator scrollable on their own?

Reply:
No but I would abstract if there's >= 3 usages.